### DISCUSSION TITLE
[Feature] add an option to disable uploading when app in cellar lte network

### DISCUSSION THREAD

**Discussion by duguying** [immich-app/immich#6214] [Feature Request]
### The feature

add an option to disable uploading when app in cellar lte network

This will saving money for lte network fee. Such as  follow 

![IMG_2745](https://github.com/immich-app/immich/assets/3050360/2b1ac035-8857-4b93-a340-5604bcbe273d)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by zackpollard** [comment]
We already support not uploading when not on WiFi, is this different somehow?

**Reply by duguying** [reply]
Oh really? That's great how to set it

**Reply by zackpollard** [reply]
It's in the settings cog on the backup page of the app. Where you turn on background backup you can change what situations it will backup in.

**Reply by zackpollard** [reply]
![Screenshot_20240106-140256.png](https://github.com/immich-app/immich/assets/1313578/0757cb12-2f5b-496f-b319-fc188e473f59)

**Reply by duguying** [reply]
Perhaps I am using an older version 。app is 1.91.4 in iOS ，without this option

**Reply by martabal** [reply]
I think that's an android feature.

Isn't it only for the background tasks ?
If you open the app with a LTE connection, does it upload your assets ?

**Reply by zackpollard** [reply]
I thought it was available for iOS background tasks too. If you have foreground uploading enabled in the setting and you open the app. Yes it will upload regardless of those settings. If you don't want that you can disable foreground upload and it'll only upload when you click the button

**Reply by duguying** [reply]
this option is now supported in iOS version 1.95.0 build.139. thanks a lot.
